src/analysis/retail/warlock/demonology/modules/talents/DreadCalling.tsx

```diff
@@ -0,0 +1,68 @@
+import { formatPercentage } from 'common/format';
+import SPELLS from 'common/SPELLS';
+import { TALENTS_WARLOCK } from 'common/TALENTS';
+import { TooltipElement } from 'interface';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events, { CastEvent } from 'parser/core/Events';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import TalentSpellText from 'parser/ui/TalentSpellText';
+
+const DAMAGE_BUFF_PER_STACK = 0.04;
+
+/**
+ * Each Soul Shard spent on Hand of Gul'dan increases the damage of your next Call Dreadstalkers by 4%.
+ */
+
+class DreadCalling extends Analyzer {
+  _callDreadstalkersCasts: number = 0;
```

Comment:
More of a note than anything, but prefer declaring these as `private` instead of prefixing them with `_` if they're not intended to be used outside of the class.

Reply:
Thanks for the tip @ToppleTheNun, I didn’t know about ‘private’ keyword.
However The ‘_’ here serves to avoid duplicate name between class attribute and getter method. Would you have an idea on how I can avoid it otherwise?

Reply:
No really strong opinions here, but maybe something like "consumingCasts" and "buffStacksConsumed". The references to which buff and which spell consumed it should be able to be inferred by reading the analyzer.

Reply:
I renamed the attributes to remove "_". To avoid duplication with getter methods names, I made attribute names more concise, while keeping the getters names more explicit, as they are part of class' API and what they do should be clear from their name without having to implementation.

Reply:
Could you let me know if you have other feedback or merge the PR otherwise?